index.bs

```diff
@@ -2778,6 +2778,52 @@ enum ChannelInterpretation {
 </table>
 </div>
 
+<h4 id="AudioNode-tail">AudioNode tail-time</h4>
+
+An {{AudioNode}} can have a <dfn>tail-time</dfn>. This means that even when the
+{{AudioNode}} is fed silence, non-silent output can be output.
+
+{{AudioNode}}s have a non-zero tail-time if they have internal processing state
+such that input received at one time affects the output at later times. Such
+{{AudioNode}}s that have received non-silent input continue to produce
+non-silent output for the length of the tail-time after the input becomes
+silent.
+
+<h4 id="AudioNode-actively-processing">AudioNode lifetime</h4>
+
+{{AudioNode}} can be <dfn id="actively-processing">actively processing</dfn>
+during a <a>render quantum</a>, if any of the following conditions hold.
+
+- An {{AudioScheduledSourceNode}} is [=actively processing=] if and only if it
+	is playing for at least part of the current rendering quantum.
+- A {{MediaElementAudioSourceNode}} is [=actively processing=] if and only if its
+	{{MediaElementAudioSourceNode/mediaElement}} is playing for at least part of
+	the current rendering quantum.
+- A {{MediaStreamAudioSourceNode}} or {{MediaStreamTrackAudioSourceNode}} when
```

Comment:
"A MediaStreamAudioSourceNode or MediaStreamTrackAudioSourceNode when ..."

This reads a bit weird. Can you take a look once more?